features/leaveroom/impl/src/main/kotlin/io/element/android/features/leaveroom/impl/LeaveRoomPresenter.kt

```diff
@@ -78,7 +80,8 @@ private suspend fun showLeaveRoomAlert(
         val roomInfo = room.roomInfoFlow.first()
         confirmation.value = when {
             roomInfo.isDm -> Dm(roomId)
-            !roomInfo.isPublic -> PrivateRoom(roomId)
+            // If unknown, assume the room is private
+            roomInfo.isPublic.orFalse() -> PrivateRoom(roomId)
```

Comment:
There is a problem here, I think the correct test should be

```suggestion
            roomInfo.isPublic == null || !roomInfo.isPublic -> PrivateRoom(roomId)
```

I think this is not equivalent of what you've done, for the case of `roomInfo.isPublic == true`

Reply:
or `roomInfo.isPublic.orFalse().not()`